[2025-01-11T12:18:54.033+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-11T12:18:54.071+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: topcv_scraper.extract_job_data manual__2025-01-11T09:56:00.063370+00:00 [queued]>
[2025-01-11T12:18:54.080+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: topcv_scraper.extract_job_data manual__2025-01-11T09:56:00.063370+00:00 [queued]>
[2025-01-11T12:18:54.081+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-11T12:18:54.094+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_job_data> on 2025-01-11 09:56:00.063370+00:00
[2025-01-11T12:18:54.103+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2028) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-11T12:18:54.111+0000] {standard_task_runner.py:72} INFO - Started process 2029 to run task
[2025-01-11T12:18:54.106+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'topcv_scraper', 'extract_job_data', 'manual__2025-01-11T09:56:00.063370+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/crawl_data.py', '--cfg-path', '/tmp/tmppmbq6di4']
[2025-01-11T12:18:54.116+0000] {standard_task_runner.py:105} INFO - Job 7: Subtask extract_job_data
[2025-01-11T12:18:54.238+0000] {task_command.py:467} INFO - Running <TaskInstance: topcv_scraper.extract_job_data manual__2025-01-11T09:56:00.063370+00:00 [running]> on host 57959064be06
[2025-01-11T12:18:55.447+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='topcv_scraper' AIRFLOW_CTX_TASK_ID='extract_job_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-11T09:56:00.063370+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-11T09:56:00.063370+00:00'
[2025-01-11T12:18:55.448+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-11T12:22:12.097+0000] {logging_mixin.py:190} INFO - An error occurred: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
[2025-01-11T12:22:12.099+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-11T12:22:12.107+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-11T12:22:12.108+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=topcv_scraper, task_id=extract_job_data, run_id=manual__2025-01-11T09:56:00.063370+00:00, execution_date=20250111T095600, start_date=20250111T121854, end_date=20250111T122212
[2025-01-11T12:22:12.150+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-11T12:22:12.162+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-11T12:22:12.164+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
